src/locale/locales/np/messages.po

```diff
@@ -0,0 +1,9073 @@
+msgid ""
+"POT-Creation-Date: 2024-12-04 15:45+0300\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: np\n"
+"Project-Id-Version: Nepali localization for bluesky-social-app\n"
+"Report-Msgid-Bugs-To: Divyaswor Makai Shrestha <makaidivyaswor@gmail.com>\n"
+"PO-Revision-Date: \n"
+"Last-Translator: Divyaswor Makai Shrestha <makaidivyaswor@gmail.com>\n"
+"Language-Team: divyaswormakai\n"
+"Plural-Forms: \n"
+
+#: src/screens/Messages/components/ChatListItem.tsx:130
+msgid "(contains embedded content)"
+msgstr "सामग्री समावेश गरिएको छ"
+
+#: src/screens/Settings/AccountSettings.tsx:57
+#: src/view/com/modals/VerifyEmail.tsx:150
+msgid "(no email)"
+msgstr "इमेल छैन"
+
+#: src/view/com/notifications/FeedItem.tsx:232
+#: src/view/com/notifications/FeedItem.tsx:327
+#~ msgid "{0, plural, one {{formattedCount} other} other {{formattedCount} others}}"
+#~ msgstr "{0, plural, one {{formattedCount} अन्य} अन्य {{formattedCount} अन्यहरू}}"
```

Comment:
A tip for the future: the lines starting with `#~` are obsolete strings that are no longer used in the app, so you can safely ignore them and there's no need to translate them. Some translators like to remove them to keep things tidy, but there's no need to do this if you'd prefer not to, as they'll simply be ignored by the app.

Reply:
Since, they have been translated, should we just let it be for now?

Reply:
Yep, you can leave them as they are, that's no problem, I just wanted to let you know for the future :)